tests/acceptance/bootstrap/WebDav.php

```diff
@@ -1635,30 +1633,28 @@ public function uploadFileWithHeaders(
 		string $destination,
 		?array $headers = [],
 		?int $noOfChunks = 0
-	):void {
-		$chunkingVersion = $this->chunkingToUse;
+	): ResponseInterface {
+		$doChunkUpload = true;
 		if ($noOfChunks <= 0) {
-			$chunkingVersion = null;
-		}
-		try {
-			$this->pauseUploadDelete();
-			$this->response = UploadHelper::upload(
-				$this->getBaseUrl(),
-				$this->getActualUsername($user),
-				$this->getUserPassword($user),
-				$source,
-				$destination,
-				$this->getStepLineRef(),
-				$headers,
-				$this->getDavPathVersion(),
-				$chunkingVersion,
-				$noOfChunks
-			);
-			$this->lastUploadDeleteTime = \time();
-		} catch (BadResponseException $e) {
-			// 4xx and 5xx responses cause an exception
-			$this->response = $e->getResponse();
+			$doChunkUpload = false;
 		}
+
+		$this->pauseUploadDelete();
+		$response = UploadHelper::upload(
+			$this->getBaseUrl(),
+			$this->getActualUsername($user),
+			$this->getUserPassword($user),
+			$source,
+			$destination,
+			$this->getStepLineRef(),
+			$headers,
+			$this->getDavPathVersion(),
+			$doChunkUpload,
+			$noOfChunks,
+			false,
+		);
```

Comment:
false is the default one
```suggestion
			$noOfChunks,
		);
```